Assets/Scripts/PlayerControls.cs

```diff
@@ -97,6 +99,10 @@ void OnCollisionEnter(Collision collision)
 			Debug.Log("Win tIle");
 			_levelInfo.CalculateInterval(DateTime.Now);
 			GameWinPanelDisplay();
+			hasWon = true;
```

Comment:
As of now this variable does nothing.

The idea was to also add in a variable that stores if the user wins the level or not. Perhaps it would be useful in analytics. If in the future we need to deactivate powerups after the user wins etc. Should I remove it?